src/client/Microsoft.Identity.Client/Internal/TelemetryTokenTypeConstants.cs

```diff
@@ -14,10 +14,13 @@ internal static class TelemetryTokenTypeConstants
         /// Ssh-cert token type for telemetry.
         public const int SshCert = 3;
 
-        /// External token type for telemetry. 
-        public const int External = 4;
+        /// Token type for legacy AT POP
+        public const int AtPop = 4;
+
+        /// External token type for telemetry.
+        public const int External = 5;
```

Comment:
Sorry, i was not clear before. External is not a thing. It's just an implementation detail. Pls just use "AtPOP". 

AT_POP = 4
Extension1 = 5

Reply:
Seems like it was added here though, what should this be?
Bearer?
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/commit/1fd3cf51b5f86b1e9806f4797113b41fee30e99e#diff-c5ed67e6f8f1786dedc066d4c2dda88463262c54db8681f9ce7e65a6cb234e1eR22

Reply:
We put "bearer" on the wire. But the token is legacy POP.

Reply:
ok